draft-ietf-jsonpath-base.md

```diff
@@ -1069,7 +1069,10 @@ the current node is selected if and only if the expression yields true.
 
 The current node is accessible via the current node identifier `@`.
 
-An existence expression may test the result of a function expression (see {{fnex}}).
+An existence expression either tests the existence of a node
+designated by an embedded query (see {{extest}}) or tests whether the
+result of a function expression of type `Boolean` or `OptionalBoolean`
+(see {{fnex}}) is `true`.
 
```

Comment:
The above sentence doesn't cover existence tests of function return values of type `OptionalNodes` or one of its subtypes. I think the solution to that is to beef up the extest section to cover such function return values.

Reply:
Yes.  Will update PR later today.

Reply:
... and I resolved the conflict by merging main.